koku/masu/database/trino_sql/reporting_ocpawscostlineitem_daily_summary.sql

```diff
@@ -781,6 +915,15 @@ SELECT uuid(),
     region,
     unit,
     usage_amount,
+    CASE
+        WHEN upper(data_transfer_direction) = 'IN' THEN usage_amount
+        ELSE 0
+    END AS infrastructure_data_in_gigabytes,
+    CASE
+        WHEN upper(data_transfer_direction) = 'OUT' THEN usage_amount
+        ELSE 0
+    END AS infrastructure_data_out_gigabytes,
+    data_transfer_direction,
```

Comment:
Just checking cause I'm not sure, are the units for the AWS data always in `GB`?

Reply:
I looked at some real data and `pricing_unit` is always `GB` for data transfer related fields.